src/themes/form/form.js

```diff
@@ -0,0 +1,44 @@
+import tw from "twin.macro";
+
+const fullCol = tw`col-span-6`
+const halfCol = tw`col-span-6 sm:col-span-3`
+const thirdCol = tw`col-span-6 sm:col-span-3 lg:col-span-2`,
+const twoThirdsCol = tw`col-span-6 sm:col-span-4`
```

Comment:
I assume this is the best way to do this -- please change if there is a better way.